File: Luk/app/src/main/java/zilic/luk/Luk.java
package zilic.luk;

import java.util.Date;

public class Luk {

    private int sifra;
    private int slojevi;
    private Date vrijeme;
    private float mlSuza;

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public int getSlojevi() {
        return slojevi;
    }

    public void setSlojevi(int slojevi) {
        this.slojevi = slojevi;
    }

    public Date getVrijeme() {
        return vrijeme;
    }

    public void setVrijeme(Date vrijeme) {
        this.vrijeme = vrijeme;
    }

    public float getMlSuza() {
        return mlSuza;
    }

    public void setMlSuza(float mlSuza) {
        this.mlSuza = mlSuza;
    }
}

File: API-luk/src/Zilic/Luk.php
<?php
namespace Zilic;

use stdClass;
use Faker\Factory;

class Luk
{
    public function getLuka()
    {
        $faker = \Faker\Factory::create();
        $luka=[];
        
        for($i=1;$i<=17;$i++){
            $l = new stdClass();
            $l->sifra=$i;
            $sloj = rand(11,15);
            $l->slojevi= $sloj;

            $int= rand(1577836890,1577836950);
            $minute = date("i:s",$int);
            
            $l->vrijeme=$minute;
            $suze = rand(25,35)/10;
            $l->mlSuza=$suze;

            $luka[]=$l;
        }
        return $luka;
    }
}